pkg/karmadactl/version.go

```diff
@@ -0,0 +1,32 @@
+package karmadactl
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/karmada-io/karmada/pkg/version"
+	"github.com/spf13/cobra"
+)
```

Comment:
Could you please group the import by `std` --> `third-party` -->`karmada`?

Reply:
> Could you please group the import by `std` --> `third-party` -->`karmada`?

Of course, thank you for reviewing.

Reply:
Thanks, just a reminder, you need to sign the DCO[https://github.com/apps/dco] in your commit log. :)